### PR TITLE
[ i18n sprint ] new translations for fixed new individual form

### PR DESCRIPTION
**Pull request by litvinovg**
Translations for new key used in 
[VIVO PR](https://github.com/vivo-project/VIVO/pull/3776)
[Vitro PR](https://github.com/vivo-project/Vitro/pull/332)

### CODE REVIEW

**Review by bkampe — CHANGES_REQUESTED** [review]
german translation would work, but "Erstellen eines Objekts der Art" is better.

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 1]
```diff
@@ -747,7 +747,7 @@ all_capitalized = Alle
 template_capitalized = Vorlage
 selected_page_content_type = Ausgewählter Inhaltstyp für die zugehörige Seite
 
-create_new = Erstellen Sie ein neues
+create_new_individual_of_the_following_type = Erstellen der folgenden Art von Objekt
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
create_new_individual_of_the_following_type = Erstellen eines Objekts der Art
```

**Review by chenejac — APPROVED** [review]
Ok for Serbian. Please accept suggestion from Benjamin Kampe.

**Review by ghost — APPROVED** [review]
This is a blocker for PRs in templates using these new i18n properties.

**Pull request merged**